<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxForm</class>
 <widget class="QWidget" name="BoxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>759</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>831</width>
    <height>759</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>831</width>
    <height>759</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оптовые Упаковки</string>
  </property>
  <widget class="QPushButton" name="backPB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>720</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>720</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TW">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>771</width>
     <height>621</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>640</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QComboBox" name="productBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceSB">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>9999999999.989999771118164</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="sizeSB">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>640</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="priceLbl">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>650</y>
     <width>40</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>650</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кол</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="unitBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>640</y>
     <width>73</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="unitLbl">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ед. изм.</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
